### ISSUE TITLE
TypeError: Cannot read property 'exif' of undefined

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-photos-v2#173]
```
TypeError: Cannot read property 'exif' of undefined
    at /usr/local/cozy/apps/photos/build/server/models/photo.js:85:22
    at gm.<anonymous> (/usr/local/cozy/apps/photos/build/server/helpers/thumb.js:23:16)
    at gm.emit (events.js:106:17)
    at gm.<anonymous> (/usr/local/cozy/apps/photos/node_modules/gm/lib/getters.js:129:14)
    at cb (/usr/local/cozy/apps/photos/node_modules/gm/lib/command.js:301:16)
    at ChildProcess._spawn.proc.on.onExit (/usr/local/cozy/apps/photos/node_modules/gm/lib/command.js:283:9)
    at ChildProcess.emit (events.js:98:17)
    at maybeClose (child_process.js:766:16)
    at Process.ChildProcess._handle.onexit (child_process.js:833:5)
```

**Comment by clochix** [comment]
Should have already been fixed as a side effect of c7a0d6b37a9597f5ce2c9043724369f15829ffeb

**Comment by clochix** [comment]
FTR, the error message is Error: `Command failed: identify.im6: no decode delegate for this image format`/tmp/magick-CEVtFBtB' @ error/constitute.c/ReadImage/544`

Here's the binary attached to the photo:

```
{"_id":"7a30e0c61a0e98344c3cbcc0d70383ba","_rev":"2-d1c3b0cd0d11cb11fa3a1caf44e567fc","docType":"Binary","_attachments":{"raw":{"content_type":"application/octet-stream","revpos":2,"digest":"md5-iR6ZQHzyPuZl3MciAXmngw==","length":1730,"stub":true}}}
```
